.chloggen/chore_update-mongodb-driver-library.yaml

```diff
@@ -0,0 +1,28 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: mongodbreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Updated the mongo-driver to v2 
```

Comment:
Are there any user-facing changes here? If not, we can omit the changelog.

Reply:
The only thing that might impact Otel users is if a user is still using this component with mongodb 3.6 as that is supposedly getting deprecated. 

It isn't in our supported version matrix so I'm happy removing the changelog!